cmd/cdc/cli/cli_changefeed_create_test.go

```diff
@@ -126,7 +126,7 @@ func TestChangefeedCreateCli(t *testing.T) {
 	cmd := newCmdCreateChangefeed(f)
 	dir := t.TempDir()
 	configPath := filepath.Join(dir, "cf.toml")
-	err := os.WriteFile(configPath, []byte("enable-sync-point=true\r\nsync-point-interval='20m'"), 0o644)
+	err := os.WriteFile(configPath, []byte("enable-sync-point=false\r\nsync-point-interval='20m'"), 0o644)
```

Comment:
why change this? looks not related to the keyspace change.

Reply:
This is a unit test related to Kafka, but Kafka does not support sync-points. It also fails in the master branch. So I fixed it.